blog.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: blog
-title: "Quarkus - Blog"
+title: "Blog"
```

Comment:
I'm unsure this is a good change given we won't have `Quarkus` in the `<title>` anymore.